viz_scripts/scaffolding.py

```diff
@@ -105,6 +115,30 @@ def expand_userinputs(labeled_ct):
     disp.display(expanded_ct.head())
     return expanded_ct
 
+def expand_inferredlabels(inferred_ct):
+    if len(inferred_ct) == 0:
+        return inferred_ct
+
+    max_labels_list = []
+    for _, row in inferred_ct.iterrows():
+        # In the trip, prioritize availabilty of user_input over inferred_labels for label selection
+        if row.user_input == {}:
+            # Extract the label which has highest "p" value
+            max_entry = max(row.inferred_labels, key=lambda x: x['p'])
+            if (max_entry['p'] > row.confidence_threshold):
+                max_labels_list.append(max_entry['labels'])
+            else:
+                max_labels_list.append({})
+        else:
+            max_labels_list.append(row.user_input)
+
+    inferred_only_labels = pd.DataFrame(max_labels_list, index=inferred_ct.index)
```

Comment:
If this is really inferred / user labels now, I think it might be best to rename it

Reply:
Renamed it to `labeled_inferred_labels`